### ISSUE TITLE
Not updated info in event modal when using customComponents

### ISSUE THREAD

**Issue by Feduch** [schedule-x/schedule-x#329]
**Context:**

- Browser chrome
- Browser-version 123.0.6312.105
- Schedule-X version 1.28.0

**Describe the bug**

When using customComponents on a modal event, the information in the modal window is not updated when other events are clicked.
```
<script setup lang="ts">
const customComponents = {
  eventModal: CustomEventModal
}
</script>

<template>
  <div class="bg-white p-[16px] rounded-[12px]">
    <ScheduleXCalendar :calendar-app="calendarApp" :custom-components="customComponents" />
  </div>
</template>
```

```
CustomEventModal.vue

<template>
  <div class="event">
     {{ calendarEvent }}
  </div>
</template>

<script lang="ts" setup>
const props = defineProps({
  calendarEvent: {
   Object as PropType<{ title: string; id: number | string }>,
    required: true,
  },
})

</script>

<style scoped>
</style>
```

**Comment by tomosterlund** [comment]
Thanks for reporting this.

I'll look into it :)

**Comment by tomosterlund** [comment]
This is fixed in @schedule-x/calendar v1.29.0. Releases for the Vue and React components will follow too.

Thanks again for caring to report 😊

**Comment by Feduch** [comment]
Thanks a lot! Works great). Very cool project you are doing
